buildscripts/semantic-convention/templates/SemanticAttributes.java.j2

```diff
@@ -166,13 +166,13 @@ public final class {{class}} {
   public static final AttributeKey<String> HTTP_HOST = stringKey("http.host");
 
   /**
-   * @deprecated This item has been removed as of 1.13.0 of the semantic conventions.
+   * @deprecated This item has been removed as of 1.13.0 of the semantic conventions. Please use {@link SemanticAttributes.NET_SOCK_PEER_ADDR} instead.
```

Comment:
@jkwatson I briefly brought up the question of how long we retain deprecated semantic attributes [here](https://github.com/open-telemetry/opentelemetry-java/pull/4766#discussion_r977606900). If we go with one release cycle, then all of these will be deleted and this change is unnecessary.

Reply:
> Is there any harm in keeping these around longer?

AFAIK no, not really. I think we could keep them for a bit longer if it makes it easier for people to migrate off of them; we're certainly no longer using these in the instrumentation repo.

Reply:
Two versions? Three versions?

Reply:
no idea. Let's raise it as a topic at the Thursday meeting.

Reply:
added to agenda